refactor/runner/openapi.yaml

```diff
@@ -0,0 +1,168 @@
+openapi: 3.0.2
+info:
+  title: Covalent Runner Service API
+  version: 0.1.0
+paths:
+  /api/v0/workflow/{dispatch_id}/task:
+    post:
+      tags:
+        - Task
+      summary: Run Task
+      description: API Endpoint (/api/workflow/task) to run tasks
+      operationId: run_task_api_v0_workflow__dispatch_id__task_post
+      parameters:
+        - required: true
+          schema:
+            title: Dispatch Id
+            type: string
+          name: dispatch_id
+          in: path
+      requestBody:
+        content:
+          application/json:
+            schema:
```

Comment:
The doc should include a more robust example, or multiple examples (something more than just `[0]`), and some description of how to do this properly.

It also might be more stylistically consistent to make it
```
{
    'task_ids': [int]
}
```
instead of just the array... I feel like people expect JSON to always be a dict at the top level, even if it's technically OK to have an array at the top level

Reply:
Looks like the type also changed here and now it's a string array? Is there a reason that tasks are strings instead of ints?